src/datadir-tests

```diff
@@ -10,12 +10,32 @@ use Symfony\Component\Process\Process;
 
 require_once 'vendor/autoload.php';
 
-if ($argc !== 2) {
-    print 'Please supply tests directory as first parameter' . PHP_EOL;
+if ($argc == 2 && (in_array($argv[1], ['-h', '--help', '-?']))) {
```

Comment:
Práce s argumentama v CLI by se mohla vyřešit přes Symfony/Console?
https://symfony.com/doc/current/console/input.html

Reply:
Jo, šlo by to přes ArgvInput, zkusil jsem to a z první tam něco nešlo, tak jsem to udělal takhle, abych to zbytečně nezdržoval. Jestli se ukáže, že je tam potřeba někde něco přidat ještě, tak to asi celé přehodím na SF CLI App a bude klid ;)